lib/Widgets/OutgoingTransmissionListEntry.vala

```diff
@@ -168,6 +181,12 @@ public class Drop.Widgets.OutgoingTransmissionListEntry : TransmissionListEntry
         set_progress_visible (false);
     }
 
+    private void swap_buttons () {
+        action_area.remove (cancel_button);
+        action_area.add (stop_button);
+        action_area.show_all ();
```

Comment:
you haven't done a `show_all` at the same place in IncomingTransmissionListEntry.